arelle/plugin/validate/NL/rules/fg_nl.py

```diff
@@ -92,3 +97,38 @@ def rule_fg_nl_04(
             beforeName=qname(beforeName),
             afterName=qname(afterName),
         )
+
+
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[
+        DISCLOSURE_SYSTEM_NT16,
+        DISCLOSURE_SYSTEM_NT17,
+        DISCLOSURE_SYSTEM_NT18,
+    ],
+)
+def rule_fg_nl_11(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation] | None:
+    """
+    FG-NL-11: XBRL instance file Should only contain standard units based on the XBRL units Registry.
+    """
+    modelXbrl = val.modelXbrl
+    customUnits = []
+    for unitId, unit in modelXbrl.units.items():
+        for measures in unit.measures:
+            for measure in measures:
+                if measure.namespaceURI not in VALID_UTR_NAMESPACES:
```

Comment:
This seems to allow `unitRef="iso4217:brett"`, etc.

If the conclusion is to use a simple namespace-based test, it seems like we should add FIXME-style comment saying that loading from some specific version of the UTR would be more precise.